_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
why not just use the default settings instead of enforcing an advanced usage?
```
{% if site.gems contains 'jekyll-seo-tag' %}
  {% seo %}
{% endif %}
```